server/sonar-process-monitor/src/main/java/org/sonar/application/cluster/AppStateClusterImpl.java

```diff
@@ -30,6 +30,7 @@
 import org.sonar.application.AppStateListener;
 import org.sonar.application.config.AppSettings;
 import org.sonar.process.ProcessId;
+import org.sonar.process.ProcessProperties;
```

Comment:
There are two commits one for UUID and the other for the cluster. Do you mean adding a third commit ?